Description: Upstream changes introduced in version 1.2-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 betaradio (1.2-1) unstable; urgency=low
 .
   * Initial Release. (Closes: #617474)
 .
 The person named in the Author field signed this changelog entry.
Author: Shih-Yuan Lee (FourDollars) <fourdollars@gmail.com>
Bug-Debian: http://bugs.debian.org/617474

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- betaradio-1.2.orig/data/betaradio.desktop.in
+++ betaradio-1.2/data/betaradio.desktop.in
@@ -1,6 +1,5 @@
 [Desktop Entry]
 Categories=GTK;Application;AudioVideo;Audio;Player;
-Encoding=UTF-8
 Exec=betaradio
 Icon=@prefix@/share/pixmaps/betaradio/betaradio.png
 Type=Application
--- betaradio-1.2.orig/man/betaradio.1
+++ betaradio-1.2/man/betaradio.1
@@ -4,9 +4,13 @@
 .TH BETARADIO 1 "2011-01-04" GNU "BetaRadio User Manuals"
 .SH NAME
 betaradio \- An easy way to listen to internet radio of Taiwan.
+.SH SYNOPSIS
+.B betaradio
 .SH DESCRIPTION
 With this client you can listen to internet radio of Taiwan without opening a
 browser. It displays a icon on system tray that you can click and select the
 channel you want to listen to.
+.SH SEE ALSO
+.IR http://code.google.com/p/betaradio/
 .SH AUTHOR
 Shin-Yuan Lee (FourDollars) <fourdollars@gmail.com>
